tests/relational/test_multi_table_restore.py

```diff
@@ -215,6 +260,17 @@ def create_standin_project_artifacts(
             table_path = setup_path / f"synthetics_train_{table}.csv"
             rel_data.get_table_data(table).to_csv(table_path, index=False)
             tar.add(table_path, arcname=f"synthetics_train_{table}.csv")
+            for kind, score in [("individual", 90), ("cross_table", 91)]:
+                html_filename = f"synthetics_{kind}_evaluation_{table}.html"
+                html_path = setup_path / html_filename
+                with open(html_path, "w") as f:
+                    f.write("<html></html>")
+                tar.add(html_path, arcname=f"{html_filename}")
+                json_filename = f"synthetics_{kind}_evaluation_{table}.json"
+                json_path = setup_path / json_filename
+                with open(json_path, "w") as f:
+                    json.dump(_report_json_dict, f)
+                tar.add(json_path, arcname=f"{json_filename}")
```

Comment:
I probably didn't explain this well enough / it isn't clear enough from code (😞). For synthetics, there are two archive files we create, at the following times, with the following contents:

**synthetics_training.tar.gz**
- Created upon completion of synthetics models training (mt.train)
- Contains the CSVs used as data sources for each table's model


**synthetics_outputs.tar.gz**
- Created upon completion of synthetics record handlers + post-processing (mt.generate)
- Contains, at the root level of the archive, N subdirectories, one for each "run" (each call to mt.generate) (since you may want to train once and generate N times). Each run subdirectory contains:
  - relational report
  - all component report files (4 per table: individual and cross-table, html and json)
  - synthetic output data as CSVs
  - conditional seed CSVs (if any)

So we shouldn't add these files to the training archive in this test. (For more context: at training-complete-time, we'd only have one set of those files available—the artifacts downloaded from the model. Which "kind" those artifacts are (individual or cross-table) depends on the strategy used.)